recipes/quantization.md

```diff
@@ -1,13 +1,43 @@
 # Quantization and Sparsity
 
-torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) for more details.
+torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
 
 #### Quantize
 To quantize a model (default is int4 weight only quantization):
 ```
 tune run quantize --config quantization
 ```
 
+#### Quantization-Aware Training (QAT)
+
+(PyTorch 2.4+)
+
+QAT refers to applying fake quantization to weights and/or activations during finetuning,
+which means simulating only the quantization math without actually casting the original
+dtype to a lower precision.
+
+```
+tune run qat_distributed --config qat
+```
+
+This produces an unquantized model in the original data type. To get an actual quantized model,
+follow this with `tune run quantize` while specifying the same quantizer in the config.
+
+The `qat_distributed` recipe is also compatible with any full finetuning config,
+as long as a QAT quantizer is specified in the config:
+
+```yaml
+# QAT specific args
+quantizer:
+  _component_: torchtune.utils.quantization.Int8DynActInt4WeightQATQuantizer
+  groupsize: 256
+```
+
+Currently only `torchtune.utils.quantization.Int8DynActInt4WeightQATQuantizer`
+is supported. This refers to int8 dynamic per token activation quantization
+combined with int4 grouped per axis weight quantization.
```

Comment:
nit: can add a link to a reference or something here

Reply:
We don't have a README or proper docs in torchao yet, will just link to the implementation for now